app/api/suggestions/configurationManager.ts

```diff
@@ -0,0 +1,190 @@
+import entities from 'api/entities';
+import entitiesModel from 'api/entities/entitiesModel';
+import { files } from 'api/files/files';
+import settings from 'api/settings/settings';
+import languages from 'shared/languages';
+import { ObjectIdSchema } from 'shared/types/commonTypes';
+import { FileType } from 'shared/types/fileType';
+import { IXSuggestionType } from 'shared/types/suggestionType';
+import { IXSuggestionsModel } from './IXSuggestionsModel';
+import { Suggestions } from './suggestions';
+
+interface ISettingsTemplate {
```

Comment:
Maybe we could make this part of the settings type to be consistent?

Reply:
Resolving as discussed